ui/apps/platform/src/Containers/Policies/PatternFly/Modal/ImportPolicyJSONModalError.tsx

```diff
@@ -44,6 +44,7 @@ function ImportPolicyJSONError({
         setDuplicateResolution({ ...duplicateResolution, [key]: value });
     }
 
+    const duplicateErrorsOnly = duplicateErrors.length > 0;
```

Comment:
hasOnlyDuplicateErrors?

Reply:
Your suggestion is better, but I'll leave it as-is for now, and fix in Boo's parent PR.